packages/assets/scss/02-molecules/_relationship-indicators.scss

```diff
@@ -158,10 +158,10 @@
 
       a {
         color: $c-primary;
-        box-shadow: inset 0 0 white, inset 0px -1px rgba($c-primary, 0.4);
+        box-shadow: inset 0 0 $c-white, inset 0px -1px $c-box-shadow-primary-40;
```

Comment:
TBH, I don't see any differences these styles make, can you please validate?  I think we can just remove them. 
http://localhost:3000/?p=molecules-relationship-indicators